lib/appsignal/utils/data_encoder.ex

```diff
@@ -31,6 +31,9 @@ defmodule Appsignal.Utils.DataEncoder do
   def encode(resource, {key, value}) when is_binary(value) do
     Nif.data_set_string(resource, key, value)
   end
+  def encode(resource, {key, value}) when is_integer(value) and value >= 9223372036854775808 do
+    Nif.data_set_string(resource, key, "bignum:#{value}")
```

Comment:
I used `bigint:` in the ruby gem: https://github.com/appsignal/appsignal-ruby/pull/234
probably a good idea to give them the same prefix.